src/common/transformations/src/transformations/common_optimizations/align_mixed_fp32_fp16_types.cpp

```diff
@@ -77,5 +77,5 @@ bool ov::pass::AlignMixedFP32FP16Types::run_on_model(const std::shared_ptr<ov::M
         is_changed |= insert_converts_after_if_needed(node);
     }
 
-    return true;
+    return is_changed;
```

Comment:
@itikhono is it correct change?

Reply:
@ilya-lavrenov this file i recently added, it's correct a change.